types/power-assert/ts3.7/index.d.ts

```diff
@@ -27,7 +27,7 @@ declare namespace assert {
     }
 
     export function fail(actual?: any, expected?: any, message?: string, operator?: string): never;
-    export function ok(value: any, message?: string): asserts value;
```

Comment:
Without these, I think there’s no need for a `ts3.7` folder, right?

Reply:
I'm keeping this folder to introduce asserts predicate again after the problems are resolved.